eureka-core/src/main/java/com/netflix/eureka/PeerAwareInstanceRegistryImpl.java

```diff
@@ -0,0 +1,749 @@
+/*
```

Comment:
All logic related to peer nodes management is taken out to PeerNodes class. No other changes.